### ISSUE TITLE
Data Science with Python : Convert Images to Pencil Sketch using OpenCV

### ISSUE THREAD

**Issue by prathimacode-hub** [girlscript/winter-of-contributing#6932]
If you had referred any resources, add them up in "DS Resources". Similarly, if you had used datasets, include them in "DS Datasets".

This issue which help in get better understanding of converting images into pencil sketch using OpenCV library in a detailed manner. 

It's created on contributors request as mentioned in Discord channel.

Domain: Computer Vision

Mentors Assigned : Peehu Saxena | Dhruv Bajaj

Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title" should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your journey ahead. 😎

### Domain

Data Science with Python

### Type of Contribution

Audio, Video, Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

**Comment by AdityaNikhil** [comment]
**Serial no**: 815
**Batch**: 19
**Name**: Aditya Nikhil
**Type**: Video
**Outline**:
1. Brief introduction to OpenCV
2. Creating a pencil sketch effect using gaussian blur and B/W image.

**Comment by prathimacode-hub** [comment]
This is created on contributors request made on discord. You can choose audio/video or any other issue to work on. Before that do the changes addressed to the PR. Thankyou. @AdityaNikhil

**Comment by prathimacode-hub** [comment]
Issue assigned to @divya-chhoriya for documentation

**Comment by AdityaNikhil** [comment]
> This is created on contributors request made on discord. You can choose audio/video or any other issue to work on. Before that do the changes addressed to the PR. Thankyou. @AdityaNikhil

@prathimacode-hub Last PR approved. Can I work on video version of this now?!

**Comment by prathimacode-hub** [comment]
Yes, you can. Issue assigned to @AdityaNikhil for video contribution
